doc/related_projects.rst

```diff
@@ -219,4 +245,39 @@ Domain specific packages
 Snippets and tidbits
 ---------------------
 
-The `wiki <https://github.com/scikit-learn/scikit-learn/wiki/Third-party-projects-and-code-snippets>`_ has more!
+**Gists**
```

Comment:
I think these gists can't really be considered projects, and a Wiki page might best be retained for them.